cli/release.go

```diff
@@ -48,11 +56,21 @@ Examples:
 		}
 	}
 	$ flynn release add -f config.json https://registry.hub.docker.com?name=flynn/slugbuilder&id=15d72b7f573b
-	Created release f55fde802170.
+	Created release 427537e78be4417fae2e24d11bc993eb.
+
+	$ flynn release show
+	ID            427537e78be4417fae2e24d11bc993eb
+	Artifact      docker+https://registry.hub.docker.com?name=flynn/slugbuilder&id=15d72b7f573b
+	ProcessTypes  echo
+	CreatedAt     2015-05-06 21:58:12.751741 +0000 UTC
+	ENV[MY_VAR]   Hello World, this will be available in all process types.
```

Comment:
Well, `flynn env` only shows the current release, whereas this can show historic releases, and you may want to see historic env vars (e.g. `SLUG_URL`).

Alternatively, we could add a `--release` flag to `flynn env`, but I think I prefer this.